components/BrandedButton/BrandedButton.vue

```diff
@@ -1,75 +1,93 @@
 <template>
   <component
     :is="as"
-    class="fr-btn rounded-full disabled:!text-neutral-500 disabled:!border-neutral-200"
-    :class="[colors, size, type]"
-    :disabled="disabled || loading"
+    class="inline-flex items-center space-x-1 rounded-full font-medium border !bg-none !no-underline"
+    :class="[colors, sizes, isDisabled ? '!opacity-50' : '']"
+    :disabled="isDisabled"
+    aria-disabled="isDisabled"
+    :role="as === 'a' ? 'link' : ''"
+    :href="isDisabled ? undefined : href"
   >
     <AdminLoader
       v-if="loading"
+      size="16"
       class="text-sm mr-1"
+      :color="color === 'primary' ? 'white' : 'primary'"
     />
     <component
       :is="icon"
       v-if="icon"
```

Comment:
What do you think of using a `v-else-if` here ? That way the button is not growing when loading

Reply:
Done in 57523ebea2949a6a1b8b4fa0355aa5dc4b899ece